### PR TITLE
TSK-1697: Fixed ambiguous domain in listValues query when joining wit…

### PR DESCRIPTION
**Pull request by holgerhagen**
…h classifications

---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [ ] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

**Comment by holgerhagen** [comment]
https://sonarcloud.io/dashboard?branch=tsk1697&id=holgerhagen_taskana

File: lib/taskana-core/src/test/java/acceptance/task/QueryTaskListValuesAccTest.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ */
+@ExtendWith(JaasExtension.class)
+class QueryTaskListValuesAccTest extends AbstractAccTest {
+
+  @WithAccessId(user = "teamlead-1")
+  @Test
+  void should_ReturnPorTypes_When_QueryingForListOfPorTypesForWorkbasket() {
+    TaskService taskService = taskanaEngine.getTaskService();
+    KeyDomain keyDomain = new KeyDomain("GPK_KSC", "DOMAIN_A");
+    List<String> tasks =
```
Comment (gitgoodjhe) [inline lib/taskana-core/src/test/java/acceptance/task/QueryTaskListValuesAccTest.java:29]: It's not really tasks that get returned here, right? Should we maybe change this to something like taskPorTypes?

File: lib/taskana-core/src/test/java/acceptance/task/QueryTaskListValuesAccTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            .listValues(TaskQueryColumnName.POR_TYPE, SortDirection.ASCENDING);
+    assertThat(tasks).containsExactly("SDNR", "VNR");
+  }
+
+  @WithAccessId(user = "teamlead-1")
+  @Test
+  void should_ReturnAttachmentClassificationNames_When_QueryingForListNames() {
+    TaskService taskService = taskanaEngine.getTaskService();
+    KeyDomain keyDomain = new KeyDomain("GPK_KSC", "DOMAIN_A");
+    List<String> tasks =
```
Comment (gitgoodjhe) [inline lib/taskana-core/src/test/java/acceptance/task/QueryTaskListValuesAccTest.java:43]: see above

File: lib/taskana-core/src/test/java/acceptance/task/QueryTaskListValuesAccTest.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+import pro.taskana.common.api.BaseQuery.SortDirection;
+import pro.taskana.common.api.KeyDomain;
+import pro.taskana.common.test.security.JaasExtension;
+import pro.taskana.common.test.security.WithAccessId;
+import pro.taskana.task.api.TaskQueryColumnName;
+import pro.taskana.task.api.TaskService;
+
+/**
+ * Acceptance test for the usecase of adding/removing an attachment of a task and update the result
```
Comment (gitgoodjhe) [inline lib/taskana-core/src/test/java/acceptance/task/QueryTaskListValuesAccTest.java:18]: Please adjust this description

**Review by gitgoodjhe — CHANGES_REQUESTED** [review]

**Review by gitgoodjhe — DISMISSED** [review]

**Review by gitgoodjhe — APPROVED** [review]

**Pull request merged**